cart.css

```diff
@@ -0,0 +1,149 @@
+@import url(main.css);
+
+.row
+{
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    justify-content: space-around;
+}
+.col-2
+{
+    flex-basis: 50%;
+    min-width: 18rem;
+}
+.col-2 img{
+    max-width: 100%;
+}
+
+.form-container
+{
+    background: var(--white);
+    width: 20rem;
+    height: 22rem;
+    position: relative;
+    text-align: center;
+    padding: 2rem 0;
+    margin: auto;
+    box-shadow: 5px 12px 20px 10px var(--boxshawdow);
+    border-radius: 10px;
+}
+
+.col-form
+{
+    margin-top: 6rem;
+}
+
+.card-Hori
+{
+    width: 37rem;
+    height: 22rem;
+    background-color:var(--background-card);
+    display: flex;
+    transition: 0.8s;
+    border-radius: 5px;
+    margin-bottom: 1.25rem;
+    margin-top: 6rem;
+    box-shadow: 5px 12px 20px 10px var(--boxshawdow);
+}
+.card-Hori:hover
+{
+    transform: scale(1.03);
+    box-shadow: 5px 5px 5px var(--boxshawdow);
+}
+.card-image-Hori
+{
+    width: 100%;
+    /* height: 22rem; */
+    margin-right: 10px;
+    position: relative;
+}
+.card-1-Hori
+{
+    width: 18.75rem;
+    height: 22rem;
+    object-fit: cover;
+}
+.tag-Hori
+{
+    color: var(--background-card);
+    position: absolute;
+    top: 5px;
+    left: 5px;
+    background-color:var(--tag);
+    font-size: 1rem;
+    font-weight: 800;
+    padding: 2px;
+    border-radius: 6px;
+}
+
+.Price-old
+{
+    text-decoration: line-through;
+    display: inline;
+    margin: 10px 5px;
+}
+.card-price
+{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+hr{
+    margin-top: 10px;
+}
+p{
+    color:var(--black);
+}
+
+.cart-c{
+    display: flex;
+    justify-content: space-between;
+    margin: 10px;
+}
+.cart-btn
+{
+    padding: 5px;
+    margin-top: 10px;
+    font-size: 13px;
```

Comment:
use rems for font size.
so, that if you change root font-size everything is taken care of automatically